docs/src/operate/index.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 3
 
 import DocCardList from '@theme/DocCardList';
 
-This section is for the device operators who operate a device running thin-edge
-and the cloud operators who manage a fleet of thin-edge devices.
+This section is for the device operators who operate a device running %%te%%
```

Comment:
An explicit page description is required here. For two reasons:

1. `%%te%%` are not interpreted in page descriptions
2. The wording of this first sentence is not appropriate for a page description.

Currently the page for the tag `Operate` is displayed as:

> __Operate Devices__
This section is for the device operators who operate a device running %%te%%

Reply:
Yes, the `description` frontmatter is used to describe the function of the page, I'll add the missing description for the operate page.